tests/gpu/misc/Dockerfile.cuda10

```diff
@@ -0,0 +1,23 @@
+FROM rocker/ml:4.0.5-cuda10.1
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+  cuda-samples-10-0 \
+  python3-dev \
+  python3-pip \
+  gcc-7 \
+  g++-7
+
+# install tensorflow 1.15.5
+RUN pip install nvidia-pyindex
+RUN pip install nvidia-tensorflow[horovod]
+
+RUN sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-7 700 --slave /usr/bin/g++ g++ /usr/bin/g++-7
+WORKDIR /usr/local/cuda-10.0/samples
+RUN make
+
+COPY ./test-gpu.sh /test-gpu.sh
+RUN chmod +x /test-gpu.sh
```

Comment:
This line may not be needed if you run `chmod +x test-gpu.sh` to make `test-gpu.sh` executable before committing.